.orchestra/support/revng-container/install.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -euo pipefail
+
+cd "$BUILD_DIR"
+UUID="$(uuidgen)"
+IMAGE_NAME="revng-container_$UUID"
+TEST_NAME=""
+
+function cleanup {
+  trap - EXIT;
+  if [ -n "${TEST_NAME-}" ]; then
+    podman kill "$TEST_NAME"
```

Comment:
`podman` is now a dependency, we should install it.

Reply:
Fixed